### ISSUE TITLE
Move to different package?

### ISSUE THREAD

**Issue by flozano** [moquette-io/netty-mqtt5-codec#1]
I think using the io.netty.codec package is confusing... can you move it to a different package so that we can use both netty's mqtt codec and this one in parallel without risk?

**Comment by andsel** [comment]
@flozano you're right, at moment I'm using a fork inside Moquette, to have at end a broker with MQTT 5 and a MQTT3/5 codec library.
